docs/3.0.0-beta.x/migration-guide/migration-guide-beta.19-to-beta.20.md

```diff
@@ -208,6 +217,9 @@ var models = {
 
 Finally you can load this script in your mongo shell and run it.
 
+Note that after migration the `name` field of the files you uploaded will be replaced with a name without an extension.
```

Comment:
This was reverted recently to leave the extension of the original file in the name by default.

Reply:
I'm not sure this PR is still necessary

Reply:
@alexandrebodin Ok I just found #5971 extension is back to filename. But still, this can be a breaking change for some people(includes me, my frontend was broken with this change) until the new release comes out. So I think this PR is still necessary, and I think it's also necessary to let people know more about the changes when release come out. I know you Strapi team is very busy, but I want you to consider why people think that the absence of a "better document" is the most pain point. (https://github.com/strapi/strapi/issues/6634#issuecomment-643156864)